packages/react-next/src/components/Pivot/Pivot.types.ts

```diff
@@ -69,6 +69,16 @@ export interface IPivotProps extends React.HTMLAttributes<HTMLDivElement> {
    */
   linkFormat?: PivotLinkFormatType;
 
+  /**
+   * Overflow behavior when there is not enough room to display all of the links/tabs
+   *
+   * none: Pivot links will overflow the container and may not be visible
+   * menu: Display an overflow menu that contains the tabs that don't fit
```

Comment:
i think we should move these comments to where we defined `PivotOverflowBehaviorType`

Reply:
IntelliSense only shows the comment from above the param itself, so it's more useful in practice to have it here. The type definition isn't even necessary, so I'll just remove that.